keps/sig-cli/2227-kubectl-default-container/README.md

```diff
@@ -179,8 +184,36 @@ metadata:
 ```
 
 ### Test Plan
+
+#### Testing Strategy
+
 Add a unit test for each command, testing the behavior with the annotation, without the annotation, and with the --container flag
 
+##### Prerequisite testing updates
+
+<!--
+Based on reviewers feedback describe what additional tests need to be added prior
+implementing this enhancement to ensure the enhancements have also solid foundations.
+-->
+
+##### Unit tests
+
+The main unit test is in package under `vendor/k8s.io/kubectl/pkg/`.
+
+- vendor/k8s.io/kubectl/pkg/cmd/exec 70%
+- vendor/k8s.io/kubectl/pkg/cmd/util/helpers.go 38.1%
+- vendor/k8s.io/kubectl/pkg/polymorphichelpers/logsforobject.go 79%
+
+See details in <https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit&include-filter-by-regex=kubectl>.
+
+##### Integration tests
+
+N/A
+
+##### e2e tests
```

Comment:
To make it GA we'll need a test, which you can add right away before we merge the KEP.

Reply:
I will work on https://github.com/kubernetes/kubernetes/pull/115046 to add an e2e test.

Reply:
Yes, the e2e must merge before the KEP